ext/opentelemetry-ext-grpc/tests/test_server_interceptor.py

```diff
@@ -87,18 +84,26 @@ def handler(request, context):
         self.assertEqual(span.name, "")
         self.assertIs(span.kind, trace.SpanKind.SERVER)
 
+        self.assertEqual(
+            span.instrumentation_info.name, opentelemetry.ext.grpc.__name__,
+        )
+        self.assertEqual(
+            span.instrumentation_info.version,
+            opentelemetry.ext.grpc.__version__,
+        )
+
     def test_span_lifetime(self):
         """Check that the span is active for the duration of the call."""
 
-        tracer_provider = trace_sdk.TracerProvider()
-        tracer = tracer_provider.get_tracer(__name__)
-        interceptor = server_interceptor(tracer)
+        interceptor = server_interceptor()
+        tracer = trace.get_tracer(__name__)
```

Comment:
I think it is preferable for unit tests to use their own local TracerProvider, as to to not interfere with other Unit tests.

Reply:
Right. I updated this change to get the tracer from `self.tracer_provider`. There are other test cases that use `trace.get_tracer` but I'd avoid updating those as part of this PR.